### DISCUSSION TITLE
github的搜索总是被不相干的结果刷屏，希望能好好处理一下

### DISCUSSION THREAD

**Discussion by mrchzh** [community/community#33717] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/33919096/191554839-a094ee68-d297-4f2a-981a-aad1a1f9c541.png)
github的搜索总是被无效结果刷屏，希望能好好处理一下。
有些项目它的标题起了很多很多的专业名称，然后我们在搜索的时候应该就撞库了，导致搜索结果出来了，但是实际上根本是没有关系的内容！！！这对我的搜索有很大的困扰，很多都是无效结果

**Comment by mrchzh** [comment]
![image](https://user-images.githubusercontent.com/33919096/191555745-af7821bb-6a07-47df-bb4b-c99e9bedd080.png)
随便搜索一个东西就bei被它刷屏，离谱

**Comment by mrchzh** [comment]
你们的搜索算法被人家研究透了，该进化了

**Comment by colinwm** [comment]
Have you tried GitHub's new code search, available at cs.github.com?

**Reply by mrchzh** [reply]
no，Maybe I can try。
But your search results are really terrible。
This is my search“AI作画”result
![image](https://user-images.githubusercontent.com/33919096/197927072-0025e2d1-f0f8-42fe-8715-f13ef9ce4a64.png)
Results on page 4。
Their projects are full of various professional computer words, which leads to the distortion of your search results and brings great trouble to those who really want to search for relevant projects.

**Reply by mrchzh** [reply]
You can try it  AI作画

**Comment by linkerGH** [comment]
我也觉得很烦啊，这个提示词太多了，搜什么都能搜到这些东西

**Comment by linkerGH** [comment]
所以有啥解决办法吗
